app/src/main/java/org/colourhood/instamessage/MainActivity.kt

```diff
@@ -2,11 +2,46 @@ package org.colourhood.instamessage
 
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
```

Comment:
Please create a new activity named MessageViewActivity and copy the changes you did on MainActivity to the new activity. Please revoke all the changes you did on MainActivity.